ilan/README.md

```diff
@@ -1,37 +1,21 @@
-# Building an IBM HTTP Server v8.5.5 ILAN image from binaries
+# Building an IBM HTTP Server ILAN image from binaries
 
-An IBM HTTP Server ILAN image can be built by obtaining the following binaries:
-* IBM Installation Manager binaries from [developerWorks](http://www.ibm.com/developerworks/downloads/ws/wasdevelopers/)
+An IBM HTTP Server ILAN image can be built by running one script.
 
-  IBM Installation Manager binaries:
-  * agent.installer.linux.gtk.x86_64_1.6.2000.20130301_2248.zip
+Versions available:
+8.5.5.9
+8.5.5.10
+9.0.0.0
 
-* IBM HTTP Server, IBM WebServer Plugins and IBM WebSphere Customization Tools binaries from [developerWorks](http://www.ibm.com/developerworks/downloads/ws/wasdevelopers/) / [Fix Central](http://www-933.ibm.com/support/fixcentral/)
+If you require all available versions of IHS then simply run the script buildAll with the arguments of a IBM ID and password.
+These are required for downloading the binaries as part of the installation.
 
-  IBM HTTP Server 8.5.5 binaries:
-  * was.repo.8550.ihs.ilan_part1.zip
-  * was.repo.8550.ihs.ilan_part2.zip
+If you just want to build a specific version then you can run the build script with the version you require followed by an IBM ID and password.
 
-  IBM WebServer Plugins 8.5.5 binaries:
-  * was.repo.8550.plg.ilan_part1.zip
-  * was.repo.8550.plg.ilan_part2.zip
+You are required to download a version of IBM installation manager above version 8 and place it into the /ilan/im/ directory in this repository.
```

Comment:
Capitals for "Installation Manager". No trailing slash on the directory name. Drop the "in this repository" as it's their clone of the repo, not that actual repo. Can you include the link to where they can get IM from? I'd also put the pre-req before the two paragraphs that tell you how to build otherwise there's a chance that they won't see it.